atomic_reactor/plugins/post_export_operator_manifests.py

```diff
@@ -120,6 +118,9 @@ def run(self):
         if has_operator_bundle_manifest(self.workflow):
             self._verify_csv(manifests_path)
 
+        if is_scratch_build(self.workflow):
+            return
+
```

Comment:
shouldn't we also verify that it isn't empty and raise error like in line 134 ?

Reply:
@MartinBasti

Reply:
I'd just delete this and do exact same thing for scratch builds

Reply:
Fixed